### PR TITLE
WIP: Time tracker

### PR DESCRIPTION
**Pull request by konpap99**

### CODE REVIEW

**Comment by nerdsupremacist** [comment]
Is this still a WIP?

**Review by nerdsupremacist — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -55,6 +46,7 @@ const apollo = new ApolloServer(
         playground: true,
```
Comment (nerdsupremacist) [inline src/index.ts:46]: Please don't forget to reroute the playground subscription endpoint

File: src/index.ts [hunk 2]
```diff
@@ -73,7 +68,15 @@ mongoose.connect(dbURI, { useCreateIndex: true, useNewUrlParser: true, useUnifie
     else {
         console.log('💾[database]: Connected to database successfully');
         
-        app.listen(PORT, () => {
+        server.listen(PORT, () => {
+            new SubscriptionServer({
```
Comment (nerdsupremacist) [inline src/index.ts:72]: Oh and also small thing... I don't think this is passing the authenticated user to execute...

But let's not care about that. Let's just say that subscriptions have no authentication

**Comment by konpap99** [comment]
@nerdsupremacist I saw the comments. It's still a wip because I havent figured out the payload in the subscription
I will see what I can do with auth if I have time

**Comment by Pellumbengineer** [comment]
Here what i did is calculating the avg time of the previous appointment of that and trigger the subscription when appointment is marked as done!   This is what we have stated on the assigment 2. 

Here i could not test it in a scenario with previous appointments!

@nerdsupremacist  i dont understand this issue with Graphql inspector is does not pass the check any idea here?


appFinished

**Comment by Pellumbengineer** [comment]
Together with @nerdsupremacist  we came up with a solution in order to calculate the estimatedStart time for the upcoming appointments!

timeTrackerwith Matias

**Review by nerdsupremacist — APPROVED** [review]

**Pull request merged**